lib/src/calendar.dart

```diff
@@ -20,8 +20,25 @@ enum FormatAnimation { slide, scale }
 
 /// Available day of week formats. `TableCalendar` will start the week with chosen day.
 /// * `StartingDayOfWeek.monday`: Monday - Sunday
+/// * `StartingDayOfWeek.tuesday`: Tuesday - Monday
+/// * `StartingDayOfWeek.wednesday`: Wednesday - Tuesday
+/// * `StartingDayOfWeek.thursday`: Thursday - Wednesday
+/// * `StartingDayOfWeek.thursday`: Friday - Thursday
+/// * `StartingDayOfWeek.saturday`: Saturday - Friday
 /// * `StartingDayOfWeek.sunday`: Sunday - Saturday
-enum StartingDayOfWeek { monday, sunday }
+enum StartingDayOfWeek { monday, tuesday, wednesday, thursday, friday, saturday, sunday }
+int getWeekdayNumber(StartingDayOfWeek weekday) {
```

Comment:
This should be private, so that it can't be used outside of the package. Also, I would suggest a more concise solution:
```dart
int _getWeekdayNumber(StartingDayOfWeek weekday) {
  return StartingDayOfWeek.values.indexOf(weekday) + 1;
}
```